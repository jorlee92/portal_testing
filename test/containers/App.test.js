// import { mount } from 'vue-test-utils'
// import App from '@/containers/App.vue'

describe('App', () => {
  // let wrapper
  // beforeEach(() => {
  //   wrapper = mount(App)
  // })
  // it('should render the App correctly', () => {
  //   expect(wrapper.element).toMatchSnapshot()
  // })
  xit('tests to go here', () => {})
})
