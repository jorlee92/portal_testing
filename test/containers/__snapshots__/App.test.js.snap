// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render the App correctly 1`] = `
<div
  class="app-wrapper"
>
  <header
    class="level"
  >
    <div
      class="level-left"
    >
      
    this be the header
  
    </div>
  </header>
   
  <main
    id="main"
  >
    <router-view />
  </main>
   
  <footer
    class="footer"
  >
    <div
      class="container"
    >
      <div
        class="content has-text-centered"
      >
        
  This the footer
    
      </div>
    </div>
  </footer>
</div>
`;
